src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/LoadBalancerTests.ps1

```diff
@@ -1117,4 +1117,190 @@ function Test-LoadBalancer-NicAssociationDuringCreate
         # Cleanup
         Clean-ResourceGroup $rgname
     }
+}
+
+<#
+.SYNOPSIS
+Tests creating new internal Load balancer and CRUD inbound nat pools on the load balancer
+#>
+function Test-LoadBalancerInboundNatPoolConfigCRUD-InternalLB  
+{
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $vnetName = Get-ResourceName
+    $subnetName = Get-ResourceName
+    $publicIpName = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
+    $lbName = Get-ResourceName
+    $frontendName = Get-ResourceName
+    $rglocation = "West US" 
+    $resourceTypeParent = "Microsoft.Network/loadBalancers"
+    $location = "West US" 
+    
+    try 
+    {
+        # Create the resource group
+        $resourceGroup = New-AzureRMResourceGroup -Name $rgname -Location $rglocation -Tags @{Name = "testtag"; Value = "testval"} 
+        
+        # Create the Virtual Network
+        $subnet = New-AzureRMVirtualNetworkSubnetConfig -Name $subnetName -AddressPrefix 10.0.1.0/24
+        $vnet = New-AzureRMvirtualNetwork -Name $vnetName -ResourceGroupName $rgname -Location $location -AddressPrefix 10.0.0.0/16 -Subnet $subnet
+        
+        # Create the publicip
+        $publicip = New-AzureRMPublicIpAddress -ResourceGroupName $rgname -name $publicIpName -location $location -AllocationMethod Dynamic -DomainNameLabel $domainNameLabel
+
+        $frontend = New-AzureRMLoadBalancerFrontendIpConfig -Name $frontendName -SubnetId $vnet.Subnets[0].Id
+        New-AzureRMLoadBalancer -Name $lbName -ResourceGroupName $rgname -Location $location -FrontendIpConfiguration $frontend 
+        
+        $lb = Get-AzureRMLoadBalancer -Name $lbName -ResourceGroupName $rgname
```

Comment:
Fixed in the latest commit. (https://github.com/ankurchoubeymsft/azure-powershell/commit/e6cb77ad5e8e7d3875507ab8f08772cd09d8ea83)